build.py

```diff
@@ -1115,6 +1115,17 @@ def create_dockerfile_linux(
 
 ENV LD_LIBRARY_PATH=/usr/local/tensorrt/lib/:/opt/tritonserver/backends/tensorrtllm:$LD_LIBRARY_PATH
 """
+    if FLAGS.enable_intel_gpu and ("openvino" in backends):
+        df += """
+RUN apt-get update && apt-get install -y gpg-agent wget
```

Comment:
instead of docker file change - can this be handled in the cmake / build steps for the backend?

Thought: 

Can we pass an a cmake variable to customize the build and pull this dependency in 

(https://github.com/triton-inference-server/server/blob/0a543b8a0bde62a6e45d1b549127b3aaea66a835/build.py#L2343

@tanmayv25 , @nv-kmcgill53  - this would also require some changes to how we install backends once build is complete -

Reply:
How about having a setup script as a part of the backend repository? We can decide on a standardized approach common across all the backend. The build.py can then access this setup script to install all the runtime dependencies and prepare environment to use the backend.

Reply:
The idea of plug and play backends entails building backends at one place and then copying the backend libraries (`*.so`'s) to the runtime environment with tritonserver and core.
Hence, having a script to setup env would help in all use-cases.

Reply:
will need some design work - but I like that idea